runtime/src/lib.rs

```diff
@@ -586,6 +587,18 @@ impl forum::Trait for Runtime {
     }
 }
 
+impl LockComparator<<Runtime as pallet_balances::Trait>::Balance> for Runtime {
+    fn are_locks_conflicting(new_lock: &LockIdentifier, existing_locks: &[LockIdentifier]) -> bool {
+        for lock in existing_locks {
```

Comment:
Could be simplified down to any() method.

Reply:
The section has been simplified. Also, there was a bug returning exactly the opposite of what `are_locks_conflicting` should return. Fixed.